source/includes/inkling2-reference/_curriculums.html.md

```diff
@@ -228,6 +228,30 @@ concept MyConcept(input: SimState): BrainAction {
 }
 ```
 
+#### Training Parameters
+Certain training parameters can been adjusted using a “training” clause within the curriculum statement. 
+
+```inkling2--code
+concept MyConcept(input: SimState): BrainAction {
+    curriculum {
+      training {
+        EpisodeIterationLimit: 250,
+        TotalIterationLimit: 100000
+      }
+    }
+}
+```
+
+The following parameters can be specified in a training clause. Any parameter that is unspecified will fall back on a default value provided by the platform.
+
+`EpisodeIterationLimit` controls the maximum number of iterations allowed per training episode. If a terminal condition or goal success condition is not reached before the episode iteration limit is reached, the episode will be terminated, and a new episode will begin. The specified value must be a positive integer. If not specified, a default value of 1000 is assumed.
+
+`TotalIterationLimit` controls the maximum number of iterations allowed for the concept. If this limit is reached before training termination conditions are achieved for the concept, training will stop. The specified value must be a positive integer, and the default value is 50M.
```

Comment:
I don't know if we want to get to this level of detail, but the platform won't stop exactly at this limit -- it will stop at the next "convenient" place after that. ("convenient" means "after each training batch" today). Mentioning since people have asked about this a few times recently when debugging things with very small iteration limits.